xml/System.Text.Json/JsonElement+ArrayEnumerator.xml

```diff
@@ -124,8 +124,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Advances the enumerator to the next element of the collection.</summary>
```

Comment:
It's interesting that we are using "element" which is what other enumerators use, but it is also overloaded in this context (to mean `JsonElement` since each "element" is of that type).

Reply:
If you would prefer me to change the language, please provide a suggestion and I will be happy to apply it. For now, I will mark the conversation as resolved since there is no action item.

Reply:
Was mainly talking out loud. Not worth re-wording. Looks good :)